### PR TITLE
Update READMEs and remove legacy info

### PR DESCRIPTION
**Pull request by will-moore**
This adds a few tips and a bit more info to the README of this app, to give users a better idea of what to expect and give a few tips.
Also removes some legacy info from the new app's README (on JS frameworks etc).

To test, follow updated README steps at https://github.com/will-moore/cookiecutter-omero-webapp/tree/improve_instructions

BUT use this command (instead of the one in the README) in order to point to *this* branch:

```
$ cookiecutter https://github.com/will-moore/cookiecutter-omero-webapp.git --checkout improve_instructions
```

### CODE REVIEW

File: {{cookiecutter.repo_name}}/README.md [hunk 1]
```diff
@@ -1,123 +1,35 @@
...
-To install `{{cookiecutter.repo_name}}`:
+Install `{{cookiecutter.webapp_name}}` in development mode as follows:
 
     # within your python venv:
     $ cd {{cookiecutter.repo_name}}
     $ pip install -e .
 
-You also need to add your app to the `omero.web.apps`{.interpreted-text
-role="property"} setting:
+Add the app to the `omero.web.apps`{.interpreted-text role="property"} setting:
```
Comment (pwalczysko) [inline {{cookiecutter.repo_name}}/README.md]: should this be interpreted ?
It does not read well on GitHub

**Review by pwalczysko — CHANGES_REQUESTED** [review]
one question https://github.com/ome/cookiecutter-omero-webapp/pull/2#discussion_r1492735979 otherwise lgtm

File: {{cookiecutter.repo_name}}/README.md (hunk 1 above)
Comment (will-moore) [reply {{cookiecutter.repo_name}}/README.md]: Fixed in d8e2491

**Comment by will-moore** [comment]
@pwalczysko - fixed that issue. OK to merge?
Any objections @jburel?

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**